inc/targetticket.class.php

```diff
@@ -971,6 +973,57 @@ protected function setTargetLocation($data, $formanswer) {
          case self::LOCATION_RULE_SPECIFIC:
             $location = $this->fields['location_question'];
             break;
+         case self::LOCATION_RULE_LAST_ANSWER:
+            $form_id = $formanswer->fields['id'];
+
+            // Get all answers for dropdown questions of this form, ordered
+            // from last to first displayed
+            $answers = $DB->request([
+               'SELECT' => ['answer.plugin_formcreator_questions_id', 'answer.answer', 'question.values'],
+               'FROM' => PluginFormcreatorAnswer::getTable() . ' AS answer',
+               'JOIN' => [
+                  PluginFormcreatorQuestion::getTable() . ' AS question' => [
+                     'ON' => [
+                        'answer' => 'plugin_formcreator_questions_id',
+                        'question' => 'id',
+                     ]
+                  ]
+               ],
+               'WHERE' => [
+                  'answer.plugin_formcreator_formanswers_id' => $form_id,
+                  'question.fieldtype'                       => "dropdown",
+               ],
+               'ORDER' => [
+                  'row DESC',
+                  'col DESC',
+               ]
+            ]);
+
+            foreach ($answers as $answer) {
+               // Decode dropdown settings
+               $question = PluginFormcreatorQuestion::getById($answer[PluginFormcreatorQuestion::getForeignKeyField()]);
+               $itemtype = $question->fields['itemtype'];
+
+               // Skip if not a dropdown on locations
+               if ($itemtype !== Location::class) {
+                  continue;
+               }
+
+               // Skip if question was not answered
+               if (empty($answer['answer'])) {
+                  continue;
+               }
+
+               // Skip if question is not visible
+               if (!$formanswer->isFieldVisible($answer['plugin_formcreator_questions_id'])) {
+                  continue;
+               }
+
+               // Found a valid answer, stop here
+               $location = $answer['answer'];
+               break;
```

Comment:
The feature should use the lastr valid answer. If we break; the code will select the 1st valid answer. I think this break should  be removed.

Reply:
It would if we started from the first answer, but the trick here is that we are starting from the end of the form:

![image](https://user-images.githubusercontent.com/42734840/154500591-c8187802-7d9e-4f46-84bb-1a85e15ef8a8.png)
![image](https://user-images.githubusercontent.com/42734840/154500702-f083e5e8-ac78-4267-b8ab-39be6e7b9498.png)